docs/examples/pandapower_example.ipynb

```diff
@@ -87,9 +87,11 @@
    "outputs": [],
    "source": [
     "%%capture cap --no-stderr\n",
-    "\n",
+    "import pandas as pd\n",
```

Comment:
This import might not be needed as pandapower imports pandas. Not sure if users might run cells independently.

Reply:
Still needed. But not anymore.